Resources/ServerInfo/_NF/Guidebook/Rules/6_Safezone.xml

```diff
@@ -18,4 +18,8 @@
     -- Stealing ships or ships' cargo.
     -- Selling someone else's ship you hadn't pirated outside the safe zone.
     -- Joining a crew under false pretenses with the intent to mutiny and take the ship.
+    - Players must make best efforts to keep the docks at Frontier Outpost free and clear for the use of others.
+    -- The Trade Outpost / Trade Mall can be used if you wish to remain docked to a station to conduct business.
+    -- The SR and STC cannot grant permanent or extended docking to Frontier Outpost.
+    -- Exceptions can be made for shuttles specific to outpost staff roles at the discretion of the SR or STC, but must be managed to keep docks clear at times of high demand.
```

Comment:
I would rather this be reworded.  I don't want to have to explain that "shuttles specific to outpost staff roles" doesn't include the security guard's taco ship.

Perhaps something like "Exceptions can be made for outpost staff owned ships that are necessary to fulfill their job's responsibilities.  SR or STC still has the right to have you move your ship to a different dock or undock entirely.".

Reply:
Maybe I'm misreading it or something...maybe my wording isn't any better, what do you think?

Reply:
Yeah I was trying to specifically refer to ships that can only be purchased from the staff shipyard. Perhaps I can tweak it so it reflects that instead. That might suffice...or it might not be any different! I'll give it a mull.

Reply:
"Exceptions can be made for outpost employees during low traffic times at the discretion of the SR or STC."?

Reply:
ah nevermind, don't think that is any good.